### ISSUE TITLE
            self.scheduler.step()  in () should not have epoch

### ISSUE THREAD

**Issue by everpromise** [IvoryCandy/pytorch-cifar10#10]
self.scheduler.step() 
not             self.scheduler.step(epoch)
